src/cloudant/database.py

```diff
@@ -842,6 +842,55 @@ def get_show_function_result(self, ddoc_id, show_name, doc_id):
                         headers)
         return resp.text
 
+    def update_handler_result(self, ddoc_id, handler_name, doc_id=None, **params):
+        """
+        Creates or updates a document from the specified database based on the
+        update handler function provided.  Update handlers are used, for
+        example, to provide server-side modification timestamps, and document
+        updates to individual fields without the latest revision. You can
+        provide query parameters needed by the update handler function using
+        the ``params`` argument.
+
+        Create a document with a generated ID:
+
+        .. code-block:: python
+
+            # Assuming that 'update001' update handler exists as part of the
+            # 'ddoc001' design document in the remote database...
+            # Execute 'update001' to create a new document
+            resp = db.update_handler_result('ddoc001', 'update001')
```

Comment:
I think it would be better if we had an example of calling the update handler as part of creating / updating a document, so we would provide the document body, I don't think there is much value for a user to have an example which is creating a document using just the update handle alone.